mgmt_cli_test.py

```diff
@@ -143,6 +209,37 @@ def update_config_file(self):
             self.bucket_name = f"gcs:{self.params.get('backup_bucket_location')}"
         self.is_cred_file_configured = True
 
+    def generate_background_read_load(self):
+        self.log.info('Starting c-s read')
+        stress_cmd = self.params.get('stress_read_cmd')
+        number_of_nodes = self.params.get("n_db_nodes")
+        number_of_loaders = self.params.get("n_loaders")
+
+        scylla_version = self.db_cluster.nodes[0].scylla_version
+        if "2019.1" in scylla_version:
```

Comment:
I would prefer not comparing strings, when we talk about versions...

Reply:
How should I compare them, then? 
https://github.com/scylladb/scylla-cluster-tests/blob/master/unit_tests/test_cluster.py#L208
This is how we do it in sct